ci-operator/step-registry/upi/vsphere/windows/pre/upi-vsphere-windows-pre-commands.sh

```diff
@@ -17,6 +17,18 @@ function createWindowsInstanceFile() {
       echo "$(date -u --rfc-3339=seconds) - created ${windows_instance_file}"
 }
 
+# Creates a folder at the given path if it does not exist.
+# Will return error if the parent folder also does not exist.
+# Takes the folder's path as an argument.
+function ensureSubfolderExists() {
+      folder=$1;
```

Comment:
s/folder/PATH

Reply:
I dont want to collide with the existing PATH variable